students/rob_sanchez/Lesson 2/Grid_Printer.py

```diff
@@ -0,0 +1,41 @@
+#Grid Printer exercise
+def print_grid(n):
+    #variables
+    grid_size=2
+
+    if n%2 == 0:
+        row2_size=n+1
+    else: row2_size=n
+
+    size=n/2
```

Comment:
You need to use floor division, convert to int or figure out some other way to handle remainders here because Python won't let you multiply a string times a float the `row` operations below